src/algorithm/center-of-mass.hpp

```diff
@@ -164,7 +170,7 @@ namespace pinocchio
   /// \param[in] model The model structure of the rigid body system.
   /// \param[in] data The data structure of the rigid body system.
   /// \param[in] q The joint configuration vector (dim model.nq).
-  /// \param[in] computeSubtreeComs If true, the algorithm also computes the center of mass of the subtrees.
+  /// \param[in] computeSubtreeComs If true, the algorithm also computes the center of mass of the subtrees, expressed in the world coordinate frame.
```

Comment:
Isn't this a bit of  problem, given that other functions like ```centerOfMass``` compute the CoM position in the root joint frame ? This means that ```data.com``` can hold positions in the local or in the world frame, depending on the order of function calls.

This should at least be mentioned in the documentation of Data, that currently states

``` Vector of subtree center of mass positions expressed in the root joint of the subtree. ```

Reply:
Yes, I will change the behavior of the algorithms to always express the com of the subtree in the local frame of the Joint. It will be easier, and less error-prone.